src/Settings/BankingInformationSettings/BankingInformationSettingsForm.js

```diff
@@ -0,0 +1,92 @@
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+
+import {
+  Button,
+  Col,
+  Headline,
+  Pane,
+  PaneFooter,
+  PaneHeader,
+  RadioButton,
+  RadioButtonGroup,
+  Row,
+} from '@folio/stripes/components';
+import stripesForm from '@folio/stripes/final-form';
+
+const BankingInformationSettingsForm = ({
+  handleSubmit,
+  pristine,
+  submitting,
+}) => {
+  const paneFooter = useMemo(
+    () => {
+      const end = (
+        <Button
+          id="clickable-save-contact-person-footer"
+          type="submit"
+          buttonStyle="primary mega"
+          disabled={pristine || submitting}
+          onClick={handleSubmit}
+        >
+          <FormattedMessage id="ui-organizations.settings.accountTypes.save.button" />
+        </Button>
+      );
+
+      return <PaneFooter renderEnd={end} />;
+    },
+    [handleSubmit, pristine, submitting],
+  );
+
+  const paneTitle = <FormattedMessage id="ui-organizations.settings.bankingInformation" />;
+
+  return (
+    <Pane
+      defaultWidth="fill"
+      id="banking-information"
+      renderHeader={renderProps => <PaneHeader {...renderProps} paneTitle={paneTitle} />}
+      footer={paneFooter}
+    >
+      <Row>
+        <Col xs={12}>
+          <Headline margin="none">
+            <FormattedMessage id="ui-organizations.settings.bankingInformation" />
+          </Headline>
+        </Col>
+        <Col xs={12}>
+          <Field
+            name="value"
+            component={RadioButtonGroup}
```

Comment:
Interesting, why are there radio buttons instead of the checkbox as in the other similar settings?